spec/integration/matcher_operator_data/type_object.yml

```diff
@@ -0,0 +1,25 @@
+- name: existing field - matches ObjectId
```

Comment:
I think the filename for this and `type_object_id.yml` were swapped accidentally.